{"app_name":"metabase-sync-starter","time":"2023-11-24 22:32:36.621","level":"ERROR","thread":"metabase-sync-server","class":"com.alibaba.druid.pool.DruidDataSource","message":"init datasource error, url: jdbc:postgresql://172.16.5.66:5432/metabase?currentSchema=public","stack_trace":"org.postgresql.util.PSQLException: \u5C1D\u8BD5\u8FDE\u7EBF\u5DF2\u5931\u8D25\u3002\n\tat org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:331)\n\tat org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)\n\tat org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223)\n\tat org.postgresql.Driver.makeConnection(Driver.java:402)\n\tat org.postgresql.Driver.connect(Driver.java:261)\n\tat com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1644)\n\tat com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1710)\n\tat com.alibaba.druid.pool.DruidDataSource.init(DruidDataSource.java:912)\n\tat com.boulderai.metabase.etl.tl.neo4j.service.wal.PostgresConnectionManager.checkLocalConfig(PostgresConnectionManager.java:94)\n\tat com.boulderai.metabase.etl.tl.neo4j.service.wal.PostgresConnectionManager.init(PostgresConnectionManager.java:108)\nCaused by: java.net.SocketTimeoutException: Connect timed out\n\tat java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:546)\n\tat java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:597)\n\tat java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)\n\tat java.base/java.net.Socket.connect(Socket.java:633)\n\tat org.postgresql.core.PGStream.createSocket(PGStream.java:241)\n\tat org.postgresql.core.PGStream.<init>(PGStream.java:98)\n\tat org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:109)\n\tat org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:235)\n\tat org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)\n\tat org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223)\n"}
{"app_name":"metabase-sync-starter","time":"2023-11-24 22:32:36.654","level":"ERROR","thread":"metabase-sync-server","class":"com.alibaba.druid.pool.DruidDataSource","message":"{dataSource-1} init error","stack_trace":"org.postgresql.util.PSQLException: \u5C1D\u8BD5\u8FDE\u7EBF\u5DF2\u5931\u8D25\u3002\n\tat org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:331)\n\tat org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)\n\tat org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223)\n\tat org.postgresql.Driver.makeConnection(Driver.java:402)\n\tat org.postgresql.Driver.connect(Driver.java:261)\n\tat com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1644)\n\tat com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1710)\n\tat com.alibaba.druid.pool.DruidDataSource.init(DruidDataSource.java:912)\n\tat com.boulderai.metabase.etl.tl.neo4j.service.wal.PostgresConnectionManager.checkLocalConfig(PostgresConnectionManager.java:94)\n\tat com.boulderai.metabase.etl.tl.neo4j.service.wal.PostgresConnectionManager.init(PostgresConnectionManager.java:108)\nCaused by: java.net.SocketTimeoutException: Connect timed out\n\tat java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:546)\n\tat java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:597)\n\tat java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)\n\tat java.base/java.net.Socket.connect(Socket.java:633)\n\tat org.postgresql.core.PGStream.createSocket(PGStream.java:241)\n\tat org.postgresql.core.PGStream.<init>(PGStream.java:98)\n\tat org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:109)\n\tat org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:235)\n\tat org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)\n\tat org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223)\n"}
{"app_name":"metabase-sync-starter","time":"2023-11-24 22:32:36.666","level":"ERROR","thread":"metabase-sync-server","class":"c.b.m.e.t.n.s.w.PostgresConnectionManager","message":"init dataSource  error!","stack_trace":"org.postgresql.util.PSQLException: \u5C1D\u8BD5\u8FDE\u7EBF\u5DF2\u5931\u8D25\u3002\n\tat org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:331)\n\tat org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)\n\tat org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223)\n\tat org.postgresql.Driver.makeConnection(Driver.java:402)\n\tat org.postgresql.Driver.connect(Driver.java:261)\n\tat com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1644)\n\tat com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1710)\n\tat com.alibaba.druid.pool.DruidDataSource.init(DruidDataSource.java:912)\n\tat com.boulderai.metabase.etl.tl.neo4j.service.wal.PostgresConnectionManager.checkLocalConfig(PostgresConnectionManager.java:94)\n\tat com.boulderai.metabase.etl.tl.neo4j.service.wal.PostgresConnectionManager.init(PostgresConnectionManager.java:108)\nCaused by: java.net.SocketTimeoutException: Connect timed out\n\tat java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:546)\n\tat java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:597)\n\tat java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)\n\tat java.base/java.net.Socket.connect(Socket.java:633)\n\tat org.postgresql.core.PGStream.createSocket(PGStream.java:241)\n\tat org.postgresql.core.PGStream.<init>(PGStream.java:98)\n\tat org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:109)\n\tat org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:235)\n\tat org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)\n\tat org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223)\n"}
{"app_name":"metabase-sync-starter","time":"2023-11-24 22:32:37.548","level":"ERROR","thread":"metabase-sync-server","class":"c.b.m.etl.tl.neo4j.util.DbOperateUtil","message":"queryBeanList error! sql:  select  id,table_name as tableName ,class_name as className,status,                 check_value as checkValue,check_column as checkColumn, bean_name as  beanName, syn_type as synType,      relation_name  as  relationName  ,      delete_column  as  deleteColumn ,       delete_column_value  as deleteColumnValue                         from  da_table_to_bean_config where status= 1 ","stack_trace":"java.sql.SQLException: QueryRunner requires a DataSource to be invoked in this way, or a Connection should be passed in\n\tat org.apache.commons.dbutils.AbstractQueryRunner.prepareConnection(AbstractQueryRunner.java:200)\n\tat org.apache.commons.dbutils.QueryRunner.query(QueryRunner.java:287)\n\tat com.boulderai.metabase.etl.tl.neo4j.util.DbOperateUtil.queryBeanList(DbOperateUtil.java:74)\n\tat com.boulderai.metabase.etl.tl.neo4j.service.wal.Neo4jSyncContext.initTableBeanConfig(Neo4jSyncContext.java:550)\n\tat com.boulderai.metabase.etl.tl.neo4j.service.wal.Neo4jSyncContext.initData(Neo4jSyncContext.java:408)\n\tat com.boulderai.metabase.etl.tl.neo4j.consumer.PgChangeEventConsumer.checkRabbitTemplate(PgChangeEventConsumer.java:240)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\n"}
{"app_name":"metabase-sync-starter","time":"2023-11-24 22:32:37.563","level":"ERROR","thread":"metabase-sync-server","class":"c.b.m.etl.tl.neo4j.util.DbOperateUtil","message":"queryBeanList error! sql:  select     id  ,     name  ,       code  ,      alias  ,     description,     level  ,      parent_id  as parentId ,       namespace_id  as namespaceId,      status  ,       created_at as createdAt ,       updated_at as updatedAt ,      created_by  as createdBy,     updated_by as updatedBy ,       english_name  as englishName ,     is_main_entity as mainEntity  ,    table_name as tableName ,       data_type as dataType ,       namespace_code  as namespaceCode from  da_logic_entity where status >=2 ","stack_trace":"java.sql.SQLException: QueryRunner requires a DataSource to be invoked in this way, or a Connection should be passed in\n\tat org.apache.commons.dbutils.AbstractQueryRunner.prepareConnection(AbstractQueryRunner.java:200)\n\tat org.apache.commons.dbutils.QueryRunner.query(QueryRunner.java:287)\n\tat com.boulderai.metabase.etl.tl.neo4j.util.DbOperateUtil.queryBeanList(DbOperateUtil.java:74)\n\tat com.boulderai.metabase.etl.tl.neo4j.service.wal.Neo4jSyncContext.initModelDataConfig(Neo4jSyncContext.java:429)\n\tat com.boulderai.metabase.etl.tl.neo4j.service.wal.Neo4jSyncContext.initData(Neo4jSyncContext.java:409)\n\tat com.boulderai.metabase.etl.tl.neo4j.consumer.PgChangeEventConsumer.checkRabbitTemplate(PgChangeEventConsumer.java:240)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\n"}
{"app_name":"metabase-sync-starter","time":"2023-11-24 22:32:37.579","level":"ERROR","thread":"metabase-sync-server","class":"c.b.m.etl.tl.neo4j.util.DbOperateUtil","message":"query error! sql:  SELECT    tc.table_name,   kcu.column_name  ,tc.table_schema         FROM        information_schema.table_constraints AS tc              JOIN information_schema.key_column_usage AS kcu ON tc.constraint_name = kcu.constraint_name             JOIN information_schema.constraint_column_usage AS ccu ON ccu.constraint_name = tc.constraint_name             WHERE    tc.constraint_type = 'PRIMARY KEY'  ORDER BY  tc.table_name  ","stack_trace":"java.sql.SQLException: QueryRunner requires a DataSource to be invoked in this way, or a Connection should be passed in\n\tat org.apache.commons.dbutils.AbstractQueryRunner.prepareConnection(AbstractQueryRunner.java:200)\n\tat org.apache.commons.dbutils.QueryRunner.query(QueryRunner.java:287)\n\tat com.boulderai.metabase.etl.tl.neo4j.util.DbOperateUtil.queryMapList(DbOperateUtil.java:36)\n\tat com.boulderai.metabase.etl.tl.neo4j.service.wal.Neo4jSyncContext.initModelDataConfig(Neo4jSyncContext.java:437)\n\tat com.boulderai.metabase.etl.tl.neo4j.service.wal.Neo4jSyncContext.initData(Neo4jSyncContext.java:409)\n\tat com.boulderai.metabase.etl.tl.neo4j.consumer.PgChangeEventConsumer.checkRabbitTemplate(PgChangeEventConsumer.java:240)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\n"}
{"app_name":"metabase-sync-starter","time":"2023-11-24 22:32:46.672","level":"ERROR","thread":"Druid-ConnectionPool-Create-1937889008","class":"com.alibaba.druid.pool.DruidDataSource","message":"create connection SQLException, url: jdbc:postgresql://172.16.5.66:5432/metabase?currentSchema=public, errorCode 0, state 08001","stack_trace":"org.postgresql.util.PSQLException: \u5C1D\u8BD5\u8FDE\u7EBF\u5DF2\u5931\u8D25\u3002\n\tat org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:331)\n\tat org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)\n\tat org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223)\n\tat org.postgresql.Driver.makeConnection(Driver.java:402)\n\tat org.postgresql.Driver.connect(Driver.java:261)\n\tat com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1644)\n\tat com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1710)\n\tat com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2774)\nCaused by: java.net.SocketTimeoutException: Connect timed out\n\tat java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:546)\n\tat java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:597)\n\tat java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)\n\tat java.base/java.net.Socket.connect(Socket.java:633)\n\tat org.postgresql.core.PGStream.createSocket(PGStream.java:241)\n\tat org.postgresql.core.PGStream.<init>(PGStream.java:98)\n\tat org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:109)\n\tat org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:235)\n\tat org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)\n\tat org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223)\n"}
{"app_name":"metabase-sync-starter","time":"2023-11-24 22:37:19.467","level":"ERROR","thread":"Druid-ConnectionPool-Create-1160199488","class":"com.alibaba.druid.pool.DruidDataSource","message":"create connection SQLException, url: jdbc:postgresql://172.16.5.66:5432/metabase?currentSchema=public, errorCode 0, state 08001","stack_trace":"org.postgresql.util.PSQLException: Connection to 172.16.5.66:5432 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.\n\tat org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:319)\n\tat org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)\n\tat org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223)\n\tat org.postgresql.Driver.makeConnection(Driver.java:402)\n\tat org.postgresql.Driver.connect(Driver.java:261)\n\tat com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1644)\n\tat com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1710)\n\tat com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:2774)\nCaused by: java.net.ConnectException: Operation timed out\n\tat java.base/sun.nio.ch.Net.pollConnect(Native Method)\n\tat java.base/sun.nio.ch.Net.pollConnectNow(Net.java:672)\n\tat java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:549)\n\tat java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:597)\n\tat java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)\n\tat java.base/java.net.Socket.connect(Socket.java:633)\n\tat org.postgresql.core.PGStream.createSocket(PGStream.java:241)\n\tat org.postgresql.core.PGStream.<init>(PGStream.java:98)\n\tat org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:109)\n\tat org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:235)\n"}
